plugins/vpp/l2plugin/bd_state.go

```diff
@@ -60,72 +61,77 @@ type BridgeDomainStateUpdater struct {
 }
 
 // Init bridge domain state updater.
-func (plugin *BridgeDomainStateUpdater) Init(ctx context.Context, logger logging.PluginLogger, goVppMux govppmux.API, bdIndexes l2idx.BDIndex, swIfIndexes ifaceidx.SwIfIndex,
+func (c *BridgeDomainStateUpdater) Init(ctx context.Context, logger logging.PluginLogger, goVppMux govppmux.API, bdIndexes l2idx.BDIndex, swIfIndexes ifaceidx.SwIfIndex,
 	notificationChan chan BridgeDomainStateMessage, publishBdState func(notification *BridgeDomainStateNotification)) (err error) {
 	// Logger
-	plugin.log = logger.NewLogger("-l2-bd-state")
-	plugin.log.Info("Initializing BridgeDomainStateUpdater")
+	c.log = logger.NewLogger("-l2-bd-state")
 
 	// Mappings
-	plugin.bdIndexes = bdIndexes
-	plugin.ifIndexes = swIfIndexes
+	c.bdIndexes = bdIndexes
+	c.ifIndexes = swIfIndexes
 
 	// State publisher
-	plugin.notificationChan = notificationChan
-	plugin.publishBdState = publishBdState
-	plugin.bdState = make(map[uint32]*l2.BridgeDomainState_BridgeDomain)
+	c.notificationChan = notificationChan
+	c.publishBdState = publishBdState
+	c.bdState = make(map[uint32]*l2.BridgeDomainState_BridgeDomain)
 
 	// VPP channel
-	plugin.vppCh, err = goVppMux.NewAPIChannel()
-	if err != nil {
-		return err
+	if c.vppCh, err = goVppMux.NewAPIChannel(); err != nil {
+		return errors.Errorf("failed to create API channel: %v", err)
 	}
 
 	// Name-to-index watcher
-	plugin.bdIdxChan = make(chan l2idx.BdChangeDto, 100)
-	bdIndexes.WatchNameToIdx("bdplugin_bdstate", plugin.bdIdxChan)
+	c.bdIdxChan = make(chan l2idx.BdChangeDto, 100)
+	bdIndexes.WatchNameToIdx("bdplugin_bdstate", c.bdIdxChan)
 
 	var childCtx context.Context
-	childCtx, plugin.cancel = context.WithCancel(ctx)
+	childCtx, c.cancel = context.WithCancel(ctx)
 
 	// Bridge domain notification watcher
-	go plugin.watchVPPNotifications(childCtx)
+	go c.watchVPPNotifications(childCtx)
+
+	c.log.Info("Initializing bridge domain state updater plugin")
```

Comment:
We should probably not call this plugin, since it actually isn't one. Might be confusing for someone reading the logs.